RInAppMessaging/Classes/Services/ConfigurationService.swift

```diff
@@ -70,7 +71,21 @@ extension ConfigurationService {
         )
     }
 
-    func buildURLRequest(url: URL) -> Result<URLRequest, Error> {
+    private func buildRequestHeader() -> [HeaderAttribute] {
+        let Keys = Constants.Request.Header.self
+        var additionalHeaders: [HeaderAttribute] = []
+
+        if let subId = bundleInfo.inAppSubscriptionId, !subId.isEmpty {
+            additionalHeaders.append(HeaderAttribute(key: Keys.subscriptionID, value: subId))
+        } else {
+            Logger.debug("Info.plist must contain a valid InAppMessagingAppSubscriptionID")
+            assertionFailure()
```

Comment:
It's safe to call this in production release code isn't it?

Reply:
If we release with optimization setting `-O` (which we do) then that method has no efect